app/controllers/companies_controller.rb

```diff
@@ -7,8 +7,8 @@ def index
   end
 
   def serialize_company(company)
-    serialized_socials = Social.where(company_id: company.id).map do |social|
-      social.slice(:name, :url)
+    serialized_socials = Social.where(company_id: company.id).all.each_with_object({}) do |social, acc|
```

Comment:
```suggestion
    serialized_socials = company.socials.each_with_object({}) do |social, acc|
```

What about this?

Reply:
Will fix in the other PR and rebase :smile: